mappings/net/minecraft/util/Util.mapping

```diff
@@ -205,6 +205,11 @@ CLASS net/minecraft/class_156 net/minecraft/util/Util
 		COMMENT @see #combine(List)
 		COMMENT @see #combineSafe(List)
 		ARG 0 futures
+	METHOD method_43498 waitUntilDone (Ljava/util/function/Function;Ljava/util/function/Predicate;)Ljava/lang/Object;
+		ARG 0 queuedValueSupplier
```

Comment:
```suggestion
		ARG 0 resultFactory
```
Also document this that the factory is `applyExecutor -> result`, that the async preparation tasks are supposed to be run when the factory returns the preliminary result, and the preparation tasks should send apply work to the given apply executor.

Otherwise, this would be extremely confusing. The rationale of this utility is that we thus can run tasks in the prepare-apply model when the game engine isn't in  the ticking mode yet.